extension/spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java

```diff
@@ -523,7 +470,13 @@ public static String randomBackend(SparkSettings sparkSettings , Logger logger)
             logger.error(SHOULD_NOT_HAPPEN_MESSAGE);
             throw new ShouldNeverHappenException();
         }
-        List<BackendRow> backendRows = backend.getRows().stream().filter(v -> v.getAlive()).collect(Collectors.toList());
+        List<BackendRow> backendRows = backend.stream().map(array -> {
+            BackendRow backendRow = new BackendRow();
+            backendRow.setIP(array.get(2));
```

Comment:
Why changed here? I think parsing to BackendRow is a more reasonable way than List<List<String>>. Parsing to BackendRow can eliminate hard code here.

Reply:
The same as above

Reply:
emm, have you updated code here?  I think this change should be reverted to current version.  Hard code like `array.get(2)` is unreasonable